explorer/client/src/pages/object-result/views/shared/ObjHeader.tsx

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+import { ReactComponent as ObjIcon } from '../../../../assets/SVGIcons/Call.svg';
+import GoBack from '../../../../components/goback/GoBack';
+import Longtext from '../../../../components/longtext/Longtext';
+
+import styles from './ObjHeader.module.css';
+
+type ObjHeaderData = {
```

Comment:
This kind of nested prop API can make it hard to optimize this component down the road (e.g. through `React.memo`. I'd generally recommend a flat props interface unless this data is coming directly through an API.

Reply:
This follows the approach used under Transaction Results in order to ensure consistency across the codebase. Any optimizations will need to be applied across both Transaction and Object Results Pages. Raising an issue on this point that clarifies how this optimization should work will require a separate PR.